rovercode/drivers/grovepi_chainable_rgb_leds.py

```diff
@@ -0,0 +1,50 @@
+"""
+Class for communicating with the GrovePi ultrasonic ranger.
+
+Here we treat it as a binary sensor.
+"""
+
+import os
+import logging
+import time
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+if os.getenv('DEVELOPMENT', 'false').lower() == 'true':
+    LOGGER.warning("In DEVELOPMENT mode. Using dummy.")
+    from drivers.dummy_grovepi_interface import chainableRgbLed_pattern, \
+        storeColor, chainableRgbLed_init, pinMode
+else:
+    from grovepi import chainableRgbLed_pattern, \
+        storeColor, chainableRgbLed_init, pinMode
+
+
+class GrovePiChainableRgbLeds:
+    """A module to set the GrovePi Chainable RGB LEDs."""
+
+    MODE_THIS_LED_ONLY = 0
+    port = None
+
+    def setup(self, port, count):
+        """Create a GrovePi Chainable RGB LEDs module."""
+        self.port = int(port)
+        pinMode(port, "OUTPUT")
+        time.sleep(1)
+        chainableRgbLed_init(self.port, count)
+        LOGGER.info("Setting up %s GrovePi Chainable RGB LEDs on port %s",
+                    count, port)
+
+    def set_color(self, led, red, green, blue):
+        """Set color in RGB, each ranged 0-255."""
+        if self.port is None:
+            raise RuntimeError("Must call setup before setting color.")
+
+        for component in (red, green, blue):
+            if not 0 <= component <= 255:
```

Comment:
It might be better to have the minimum and maximum here as constants as well. It's already used in two places, so if it changes at some point it would be nice to only have to change it in one place